From 006f0efb848edebc43c73a5f6a421a1696266157 Mon Sep 17 00:00:00 2001
From: Leah Woods <info@minifree.org>
Date: Mon, 19 Oct 2015 02:26:24 +0100
Subject: [PATCH] NOTFORMERGE: don't add CPU microcode on fam10h to fam15h
 (AMD)

Temporary fix for the libreboot project.

When disabling microcode updates in menuconfig, they are still
included, which means that libreboot will fail to build. This
patch fixes that.

TODO: get tpearson to fix it properly, and then abandon this patch

Change-Id: I018791e74f78c66d145225d6ddf0be324f41bc15
Signed-off-by: Leah Woods <info@minifree.org>
---
 src/cpu/amd/family_10h-family_15h/Makefile.inc | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/cpu/amd/family_10h-family_15h/Makefile.inc b/src/cpu/amd/family_10h-family_15h/Makefile.inc
index 6cd2513..eada52a 100644
--- a/src/cpu/amd/family_10h-family_15h/Makefile.inc
+++ b/src/cpu/amd/family_10h-family_15h/Makefile.inc
@@ -7,13 +7,3 @@ romstage-y += ram_calc.c
 ramstage-y += ram_calc.c
 ramstage-y += monotonic_timer.c
 ramstage-$(CONFIG_HAVE_ACPI_TABLES) += powernow_acpi.c
-
-# Microcode for Family 10h, 11h, 12h, and 14h
-cbfs-files-$(CONFIG_CPU_MICROCODE_MULTIPLE_FILES) += microcode_amd.bin
-microcode_amd.bin-file := 3rdparty/blobs/cpu/amd/family_10h-family_14h/microcode_amd.bin
-microcode_amd.bin-type := microcode
-
-# Microcode for Family 15h
-cbfs-files-$(CONFIG_CPU_MICROCODE_MULTIPLE_FILES) += microcode_amd_fam15h.bin
-microcode_amd_fam15h.bin-file := 3rdparty/blobs/cpu/amd/family_15h/microcode_amd_fam15h.bin
-microcode_amd_fam15h.bin-type := microcode
-- 
1.9.1

